cypress/e2e/course/copyCourseWithContent.feature

```diff
@@ -0,0 +1,69 @@
+@regression_test
+@stable_test
+@group-N
+@schedule_run
+Feature: Course - Copy course
+
```

Comment:
write a note as comment, that more contents (task, topic...) will be added in the course.